cmd/internal/kube/template/postman-secret.tmpl

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: akita-postman-secrets
```

Comment:
I know that I wrote 'akita-secrets' in the doc.  But looking at this, I think we should get the branding right here and call it something like postman-live-collection-agent-secrets, or otherwise avoid the 'Akita' name.